app/src/main/kotlin/com/simplemobiletools/calculator/helpers/BackgroundGetGraphTaskBuilder.java

```diff
@@ -0,0 +1,24 @@
+package com.simplemobiletools.calculator.helpers;
+
+import android.os.AsyncTask;
+
+import com.simplemobiletools.calculator.activities.DrawActivity;
+import com.simpletools.calculator.commons.helpers.Calculator;
+
+public class BackgroundGetGraphTaskBuilder {
```

Comment:
Any particular reason for a Builder pattern here?

Reply:
We used the builder previously to have a way to inject a mock. We reused the same strategy to mock the web request for the graph task

Reply:
In order to be able to use a fake a getGraphTask we had to make use of a this builder which can be superseded upon runtime.